src/test/scala/se/nimsa/dicom/streams/ModifyFlowTest.scala

```diff
@@ -430,4 +430,23 @@ class ModifyFlowTest extends TestKit(ActorSystem("ModifyFlowSpec")) with FlatSpe
       .expectValueChunk(mikeBytes)
       .expectDicomComplete()
   }
+
```

Comment:
Unrelated test written while looking for the bug root cause. It's still a valuable test so I let it stay.